blog/content/post/2021-12-12-optiplex-txt.md

```diff
@@ -0,0 +1,222 @@
+---
+title: A new source of trust for your platform - Intel TXT for Dasharo
+abstract: 'Do you trust the firmware on your system? No? Then this post is a
+           must-read for you. Get to know what Intel Trusted Execution
+           Technology (TXT) is and how it may help you securely measure and
+           attest your operating system and software running on your machine.
+           You will also hear about open source implementation of Intel TXT
+           for Ivy Bridge/Sandy Bridge platforms including Dell OptiPlex
+           7010 / 9010.'
+cover: /covers/dasharo-sygnet.svg
+author: michal.zygowski
+layout: post
+published: true
+date: 2021-12-12
+archives: "2021"
+
+tags:
+  - coreboot
+  - OptiPlex
+  - firmware
+  - measured-boot
+  - DRTM
+categories:
+  - Firmware
+  - Security
+
+---
+
+# Introduction
+
+Intel Trusted Execution Technology is a feature of Intel CPUs and chipsets to
+perform trusted measurement of the operation system software defined in Trusted
+Computing Group [D-RTM architecture specification](https://trustedcomputinggroup.org/wp-content/uploads/TCG_D-RTM_Architecture_v1-0_Published_06172013.pdf).
+Dell OptiPlex 7010 / 9010 is Intel TXT capable. All you need is a Intel TXT
+capable CPU (you may quickly check the Intel Trusted Execution Technology
+capability on [Intel ARK](https://ark.intel.com/content/www/us/en/ark.html#@Processors)
+for your processor).
+
+You may also want to read [previous blog posts about Dell OptiPlex 7010 / 9010](https://blog.3mdeb.com/tags/optiplex/).
+
+## DRTM rationale
+
+Why do even Trusted Execution Technology and DRTM exist? First of all we have
+to start with the firmware, the inevitable piece of code initializing the
+silicon and other hardware. Its responsibility is to prepare the platform to
+run operating system, but also to load it. So basically the owner of the
+firmware has control what operating system can be launched, and what is most
+important, how it can be launched. By how we should understand whether the
+operating system is launched as intended without any undesired actions, like
+malware or spyware installation under the hood. In a case the attacker has
```

Comment:
Recently I prefer use of word adversary instead of attacker

Reply:
Done